src/package.json

```diff
@@ -10,6 +10,7 @@
     "build": "node esbuild & npm run build:png && npm run build:svg && npm run copy:root",
     "build:png": "squoosh-cli --oxipng '{level: 2}' --quant '{numColors: 255}' -d ../dist/images ./client/images/*.png",
     "build:svg": "svgo -f ./client/images/ -o ../dist/images",
+    "test": "node --experimental-vm-modules ../node_modules/jest/bin/jest.js --runInBand --coverage .",
```

Comment:
Is it ok to move the test script below `copy:root`?  `copy:root` is grouped/run with the build scripts.  Perhaps it would be more obvious if I changed it instead to `build:root`, even though it's just a copy script 🤷